new-site/src/docs/elements/components/number-input/usage.mdx

```diff
@@ -0,0 +1,76 @@
+---
+slug: '/elements/components/number-input/code'
+title: 'NumberInput - Usage'
+---
+
+import { NumberInput } from 'hds-react';
+import { Link } from 'hds-react';
+
+## Usage
+
+### Example
+<NumberInput
+  helperText="Assistive text"
+  label="Total compensation"
+  unit="€"
+  defaultValue={1000}
+  style={{ maxWidth: '320px' }}
+/>
+
+### Principles
+- **A label should always be provided with a number input.**
+  - Make sure that the label is clear and concise. The user should immediately understand what number they are supposed to input.
+- HDS Number input also supports displaying an unit for the number. Displaying the unit is not mandatory. Only use it when you think it will help the user.
+    - Display the unit for <Link size="M" external href="https://en.wikipedia.org/wiki/International_System_of_Units">SI units</Link> (such as meters) and currencies.
+    - The unit does not need to be displayed when it is self-evident for the user, such as in "Number of people"
+- It is recommended to give the number input a default value. Placeholders should be avoided in number inputs.
+- HDS Number input supports visual steppers that can be set to increase and decrease the number value by a set amount.
+    - Steppers can be used when changes to the value are small or when they are intuitive to the user (e.g. from 0 to 100 with steps of 10).
+    - You should not use steppers when large value changes are expected.
+
+### Variations
+
+#### Default
+Default HDS Number input comes without stepper. The number is always inputted manually. Using this over a text input is still helpful since it has a `type=number` to aid screen readers and mobile keyboards.
+
+When applicable, a unit can be set for the input by using the `unit` prop.
+
+<NumberInput
+  helperText="Assistive text"
+  label="Total compensation"
+  unit="€"
+  defaultValue={1000}
+  style={{ maxWidth: '320px' }}
+/>
+
+#### With steppers
+Steppers can be enabled via providing a `step` property. Use this variant when value changes are small and you can determine logical amounts for an increase and a decrease step.
```

Comment:
The rendering of code properties (step in this example) does not look as clear as it was in the old documentation site:

New:

<img width="917" alt="image" src="https://user-images.githubusercontent.com/2777633/157475253-87e54502-fe6f-4b5a-9b2d-d161c883be99.png">

Old:

<img width="648" alt="image" src="https://user-images.githubusercontent.com/2777633/157475356-761ceb27-7862-406f-bf03-69e22303e644.png">

Reply:
We can probably copy-paste the old style - unless it was baked into the Docz plugin somehow.